<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Day</class>
 <widget class="QWidget" name="Day">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/resources.qrc">
    <normaloff>:/res/img/Logo1x1.png</normaloff>:/res/img/Logo1x1.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.996000000000000</double>
  </property>
  <widget class="QListWidget" name="Reminders">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>441</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TopBar">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
     <widget class="QLabel" name="dayTitle">
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>36</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>255</width>
        <height>36</height>
       </size>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>{Day}</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignRight">
     <widget class="closeBtn" name="closeBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ã—</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addReminder">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>441</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLabel" name="yearLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
  </widget>
  <widget class="QLabel" name="dayLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="otherInfo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>closeBtn</class>
   <extends>QLabel</extends>
   <header location="global">View.ExtendedClasses</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addReminder</sender>
   <signal>clicked()</signal>
   <receiver>Day</receiver>
   <slot>addReminder()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addReminder()</slot>
 </slots>
</ui>
